google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddHotelCallout.java

```diff
@@ -0,0 +1,231 @@
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.hotelads;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.HotelCalloutFeedItem;
+import com.google.ads.googleads.v2.enums.ExtensionTypeEnum.ExtensionType;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.CampaignExtensionSetting;
+import com.google.ads.googleads.v2.resources.ExtensionFeedItem;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingOperation;
+import com.google.ads.googleads.v2.services.CampaignExtensionSettingServiceClient;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemOperation;
+import com.google.ads.googleads.v2.services.ExtensionFeedItemServiceClient;
+import com.google.ads.googleads.v2.services.MutateCampaignExtensionSettingsResponse;
+import com.google.ads.googleads.v2.services.MutateExtensionFeedItemsResponse;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+
+/**
+ * Adds a hotel callout to a specific campaign
+ */
+
+ public class AddHotelCallout {
+
+    private static class AddHotelCalloutParams extends CodeSampleParams {
+
+        public static final String CALLOUT_TEXT = "--calloutText";
+        public static final String LANGUAGE_CODE = "--languageCode";
+
+        @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+        private Long customerId;
+
+        @Parameter(names = ArgumentNames.CAMPAIGN_ID, required = true)
+        private Long campaignId;
+
+        @Parameter(names = CALLOUT_TEXT, required = true)
+        private String calloutText;
+
+        // IETF BCP 47 compliant language code
+        @Parameter(names = LANGUAGE_CODE, required = true)
+        private String languageCode;
+
+    }
+
+    public static void main(String[] args){
+        AddHotelCalloutParams params = new AddHotelCalloutParams();
+        if (!params.parseArguments(args)){
+
+            // Either pass the required parameters for this example on the command line, or insert them
+            // into the code here. See the parameter class definition above for descriptions.
+            params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+            params.campaignId = Long.parseLong("INSERT_CAMPAIGN_ID_HERE");
+            params.calloutText = "INSERT_CAMPAIGN_TEXT_HERE";
```

Comment:
Should this be INSERT_CALLOUT_TEXT_HERE?

Reply:
Steven, please don't resolve the comment yet if you've not uploaded a commit that fixes it.

Reply:
Sorry, was doing it has a tracker for what I changed locally, won't do that.